### PR TITLE
Add a way to build with meson

### PR DESCRIPTION
**Pull request by thiblahute**

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -0,0 +1,25 @@
+project('hotdoc_bootstrap_theme', 'c')
```
Comment (MathieuDuponchelle) [inline meson.build]: C ? :P

Comment (thiblahute) [inline meson.build]: Yeah, I do not really have choice here, could be vala or rust otherwise !

**Comment by MathieuDuponchelle** [comment]
Merged, thanks!

File: meson.build (hunk 1 above)
Comment (MathieuDuponchelle) [inline meson.build]: `test`

**Pull request closed without merging**